docs/revanced-cli.md

```diff
@@ -1,68 +1,88 @@
-Downloading the packages
+Requirements
 ==
+
+- An Android 8+ device (to install the patched APK on)
+
+**PC:**
+- x86/x86_64 host architecture
+- [Zulu JDK 17](https://www.azul.com/downloads/?version=java-17-lts&package=jdk#zulu)
+- ADB (only required for ROOT installations)
+
+
+Preparing the packages
+==
+
+1. Follow the steps on [this page](https://github.com/inotia00/revanced-documentation/blob/main/docs/supplying-an-apk.md) to supply an APK to patch.
+
+2. Download the following packages to your PC:
 - [ReVanced CLI](https://github.com/inotia00/revanced-cli/releases/latest)
-- [ReVanced Patches](https://github.com/inotia00/revanced-patches/releases/latest)
+- [ReVanced Patches](https://github.com/inotia00/revanced-patches/releases/latest) (`.jar`file)
 - [ReVanced Integrations](https://github.com/inotia00/revanced-integrations/releases/latest)
 
+3. (Optional): For simplicity, place the APK you downloaded from APKMirror and the packages you downloaded in a folder named `revanced-extended`.
+
+
 Using ReVanced CLI (PC)
 ==
-1. Make sure your phone is connected
+
+## Non-ROOT installations
+
+**Running the CLI:**
+
+- For YouTube and Reddit, use the option '--rip-lib' to remove unwanted architectures. (e.g. `--rip-lib=x86 --rip-lib=x86_64` ...)
+- The option '-l' will show the list of available patches.
+- The option '-e' allows you to exclude patches. (e.g. `-e "Theme"` ...)
+- The option '-i' allows you to include patches. (e.g. `-i "MaterialYou"` `-i "GmsCore support"` ...)
 
 ```
-adb shell exit
+java -jar revanced-cli-all.jar patch \
+ -b revanced-patches.jar \
+ -m revanced-integrations.apk \
+ --options options.json \
+ -p \
+ -o revanced-extended.apk \
+ input.apk
 ```
 
-2-1. If you plan to use the root variant, check if you have root access
+After patching is complete, install `revanced-extended.apk` from the `revanced-extended` folder to your Android device.
+
+## ROOT installations
+
+**1. Make sure your phone is connected to ADB**
+
+**2. Check if you have root access:**
 ```
 adb shell su -c exit
 ```
 
-2-2. If you plan to use the root variant, copy the ADB device name
+**3. Copy the ADB device name:**
 ```
 adb devices
 ```
 
-2-3. If you plan to use the root variant, please install youtube (or youtube music) on your device first.
-```
-adb install -r youtube.apk
-```
-```
-adb install -r youtube-music.apk
-```
-
-3. Run the CLI
-```
-# Non-Root
-# For YouTube, use the option '--rip-lib' to remove the architecture (e.g. --rip-lib=x86 --rip-lib=x86_64 ...)
+> **If you haven't done so already, install YouTube / YouTube Music on your device:**
+> ```
+> adb install -r input.apk
+> ```
 
-java -jar revanced-cli-all.jar patch \
- -c \
- -o revanced.apk \
- -m app-release-unsigned.apk \
- -r revanced-resource-cache \
```

Comment:
It seems that the `-c` and `-r` flags were deprecated. I added the `-p` option because I figured it was the replacement.

Officially, this is its function: "Purge the temporary resource cache directory after patching."

(Purging failed for me, but I'm guessing that's an issue on my end, such as because of Windows or perms)

Reply:
yeah In the latest CLI, `-c` has been changed to `-p` and `-r` has been changed to `-t`

If you add the `-p` flag, the cache directory will be removed after the patch is finished, so it would be okay to remove `-t`

Since the old CLI is used in `latest-reddit-patch-info.md`, there is no change in flags